src/System.Net.Security/src/System/Net/NegotiateStreamPal.Unix.cs

```diff
@@ -19,11 +22,20 @@ namespace System.Net.Security
     //
     // This is part of the NegotiateStream PAL.
     //
-    internal static class NegotiateStreamPal
+    internal partial class NegotiateStreamPal
```

Comment:
Partial and static aren't in conflict.  I don't think we'd want instance members on this type, so perhaps "static partial" is what you meant?